src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -53,13 +40,10 @@ public PaxTarEntry(TarEntryType entryType, string entryName)
             : base(entryType, entryName, TarEntryFormat.Pax, isGea: false)
         {
             _header._prefix = string.Empty;
```

Comment:
Do we need to mark this as a breaking change?

Reply:
Not including atime/ctime is default behavior of other tools and we're now following that behavior.

I don't think it's likely we'll break someone by no longer setting these timestamps to UtcNow by default.

Reply:
@ericstj what do you think?

Reply:
It doesn't hurt to document the behavior changes, why we did them, and why we think _they are not_ going to be breaking.   We can use a breaking change notice to do so.  Let me get that started and tag you to have your input.